alpine/Dockerfile

```diff
@@ -52,7 +56,8 @@ RUN set -x \
 			| sort -u \
 	)" \
 	&& apk add --no-cache --virtual .irssi-rundeps $runDeps perl-libwww \
-	&& apk del .build-deps
+	&& apk del .build-deps \
+	&& rm -rf /var/cache/apk/*
```

Comment:
sowwy

On Wed, Apr 13, 2016 at 4:43 PM, Tianon Gravi <notifications@github.com>
wrote:

> In alpine/Dockerfile
> <https://github.com/jfrazelle/irssi/pull/8#discussion_r59644983>:
>
> > @@ -52,7 +56,8 @@ RUN set -x \
> >  			| sort -u \
> >  	)" \
> >  	&& apk add --no-cache --virtual .irssi-rundeps $runDeps perl-libwww \
> > -	&& apk del .build-deps
> > +	&& apk del .build-deps \
> > +	&& rm -rf /var/cache/apk/*
>
> --no-cache should make this unnecessary -- is something being left behind?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jfrazelle/irssi/pull/8/files/669c2b52e5db58804eb373901b79c8e49826eb27#r59644983>
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>